data/types/calculator/calculator.go

```diff
@@ -116,6 +123,12 @@ func (c *Calculator) Validate(validator structure.Validator) {
 	if c.CarbUnits != nil {
 		validator.String("carbUnits", c.CarbUnits).OneOf(CarbUnits()...)
 	}
+	if c.InputTime != nil {
+		c.InputTime.Validate(validator)
+	}
```

Comment:
The if with always be evaluated to true
```suggestion
	c.InputTime.Validate(validator)
```

Reply:
agree